src/executor/es/esExecutor.ts

```diff
@@ -39,5 +39,34 @@ export const esExecutor: Map<MigrationType, ExecutorFnc> = new Map([
                 body: resolvedMigration.migrate_script,
                 ...resolvedMigration.query_parameters
             })
+    ],
+    [
+        MigrationTypes.CREATE_DOCUMENT,
+        async (name, esClient, resolvedMigration) => {
+            const data = resolvedMigration.data;
+            if (!data) {
+                throw { error: '.data is required' };
+            }
+
+            const documents = Array.isArray(data) ? data : [data];
+            for (const doc of documents) {
+                if (!doc.id) {
+                    throw {
+                        error: `Missing id in document: ${JSON.stringify(doc)}`
+                    };
+                }
+
+                const data = {
+                    id: doc.id,
+                    body: doc,
+                    index: name,
+                    ...resolvedMigration.query_parameters
+                };
+
+                await esClient.postDocument(data);
```

Comment:
In this process, it is not a good idea to use postDocument if the goal is to register multiple data in a loop process!
This is not suitable for registering large amounts of data.

Internally, it is just executing the Index API, so it is correct to use the Bulk API if you want to use it for your purpose.

https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html